selfdrive/car/subaru/fingerprints.py

```diff
@@ -373,7 +373,6 @@
       b'\x00\x00c\xb7\x1f@\x10\x16',
       b'\x00\x00c\xd1\x1f@\x10\x17',
       b'\x00\x00c\xec\x1f@ \x04',
-      b'\x00\x00c\xec7@\x04',
```

Comment:
could not find this firmware for any route in the last 2 years

Reply:
added in intiial firmware dump https://github.com/commaai/openpilot/pull/23389/files#diff-80e8108f543ce92237391fd5701e16eba86c604555ca55eac698348dddb53564R327